source/Halibut.Tests/ParallelRequestsFixture.cs

```diff
@@ -17,6 +17,7 @@ namespace Halibut.Tests
 {
     public static class ParallelRequestsFixture
     {
+        [Parallelizable(ParallelScope.None)]
```

Comment:
These tests was failing locally due to the load they creates

Reply:
Did other tests also fail?

Reply:
I am not sure this is the right path, could you look into why they are failing? is it a real problem perhaps?

Is it this test that actually causes a high load? Or is it the number of concurrent tests running. Is the high load itself a problem or is something going wrong because of highload?

Reply:
Card created to investigate